customization/i3.md

```diff
@@ -9,19 +9,27 @@ redirect_from:
 - "/wiki/UserDoc/i3/"
 ---
 
-i3 installation in dom0
-========================
+# i3 installation in dom0
 
-Note that all of the work in this document is done in a Fedora based domU (could
+i3 is part of the testing repository (as of Qubes R3.1) and can be installed from there
+using the dom0 update mechanism.
+
+    $ qubes-dom0-update --enablerepo=qubes-dom0-current-testing i3
+    
+Qubes-specific configuation is available in a separate package and can be installed
+optionally. Otherwise you can write your own configuration (see below).
+
+    $ qubes-dom0-update --enablerepo=qubes-dom0-current-testing i3-settings-qubes
+    
+That's it. After logging out, you can select i3 in the login manager.
+
+## Compilation and installation from source
+
+Note that the compilation from source is done in a Fedora based domU (could
 be dispvm). The end result is always an `.rpm` that is copied to dom0 and then
 installed through the package manager.
 
-**Warning:** Installing software in dom0 is inherently risky, and the method
-described in this guide circumvents the usual security mechanisms for [updating
-software in dom0](/doc/software-update-dom0/).
```

Comment:
I'd not remove this warning, as the procedure below does not involve any source code verification.